### DISCUSSION TITLE
[Esame 11 Febbraio 2020] Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#237] [Esami - Bertini @ Informatica]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/71fa0e4a-6856-4696-959a-e139547e0f6e)

- Tratto da #203

**Comment by cornflexxx** [comment]
#203  Duplicato
